src/analysis/timeline/distributionToCred.js

```diff
@@ -49,16 +56,16 @@ export function distributionToCred(
     }
     cred[i] = new Array(intervals.length);
   }
-  if (scoringNodeIndices.length === 0) {
-    throw new Error("no nodes matched scoringNodePrefix");
-  }
 
   return ds.map(({interval, distribution, intervalWeight}) => {
     const intervalTotalScore = sum(
       scoringNodeIndices.map((x) => distribution[x])
     );
+
     const intervalNormalizer = intervalWeight / intervalTotalScore;
```

Comment:
Rather than checking if the intervalTotalScore is 0 for every single cred entry, we should do it once when we compute the normalizer.

`const intervalNormalizer = intervalTotalScore == 0 ? 0 : intervalWeight/intervalTotalScore`

Reply:
re-updated 😬